common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -188,14 +189,14 @@ class Backend(
             "currency" to receiptInfo.currency,
             "attributes" to subscriberAttributes.takeUnless { it.isEmpty() },
             "normal_duration" to receiptInfo.duration,
-            "intro_duration" to receiptInfo.introDuration,
-            "trial_duration" to receiptInfo.trialDuration,
-            "store_user_id" to storeAppUserID
+            "store_user_id" to storeAppUserID,
+            "pricing_phases" to receiptInfo.pricingPhases
         ).filterValues { value -> value != null }
 
-        val extraHeaders = /*receiptInfo.storeProduct?.price?.let { priceString ->
+        // TODO BC5 is price_string still expected? what to pass?
```

Comment:
@dalshamaa @swehner thoughts on the old price_string header?

Reply:
I couldn't find any reference to this in the backend code...

Reply:
oh,  that's good news then, we can remove it :)